code/parachain/runtime/picasso/src/xcmp.rs

```diff
@@ -240,28 +213,20 @@ impl orml_xtokens::Config for Runtime {
 	type CurrencyIdConvert = AssetsIdConverter;
 	type AccountIdToMultiLocation = AccountIdToMultiLocation;
 	type SelfLocation = topology::this::Local;
+	type MinXcmFee = ParachainMinFee;
 	type XcmExecutor = XcmExecutor<XcmConfig>;
+	type MultiLocationsFilter = Everything;
```

Comment:
I think we need to limit this at the start to the chains we want, like statemine and relay

Reply:
Likely it is something about sending tokens.  pallet owned by Acala and security was taken from here.

Reply:
It's limited anyway by the channels that have been opened ultimately. Just that defining this would give better errors to users hopefully. Anyway feel free to resolve.

Reply:
`MultiLocationsFilter` != channel. So that is need separate handling. So can improve later.

Reply:
acala does not sets this one.

Reply:
added test for case when location or asset does not exists. it is good enough.